common/device-model.cpp

```diff
@@ -2708,16 +2708,11 @@ namespace rs2
                 if (pair.first == "Firmware Version")
                 {
                     fw_version = pair.second;
```

Comment:
Please check if this parametes are used:
```
        std::string fw_version;
        std::string min_fw_version;
```

If not we can remove them ?

Reply:
done + rebase